Description: Disable some default sandboxes.
 Disable the default sandboxes for some programs
 we want to run natively.
Author: Lorenzo "Palinuro" Faletra <palinuro@parrotsec.org>
Last-Update: 2018-05-25

--- firejail-0.9.54.orig/src/firecfg/firecfg.config
+++ firejail-0.9.54/src/firecfg/firecfg.config
@@ -47,7 +47,7 @@ baobab
 basilisk
 bibletime
 bitlbee
-bleachbit
+# bleachbit
 blender
 blender-2.8
 bless
@@ -74,7 +74,7 @@ cin
 cinelerra
 clamdscan
 clamdtop
-clamscan
+# clamscan
 claws-mail
 clementine
 clipit
